README.md

```diff
@@ -257,8 +151,7 @@ INSTALLED_APPS = (
 </head>
 ```
 
-### Preload
```

Comment:
Perhaps all ### subsections below should be included in a new section "Advanced Usage" down below.